tst/EnergyPlus/unit/HeatBalFiniteDiffManager.unit.cc

```diff
@@ -233,4 +233,102 @@ TEST_F(EnergyPlusFixture, HeatBalFiniteDiffManager_adjustPropertiesForPhaseChang
     SurfaceFD.deallocate();
 }
 
+TEST_F(EnergyPlusFixture, HeatBalFiniteDiffManager_skipNotUsedConstructionAndAirLayer)
```

Comment:
Something kinda seems like it went awry here.  Possibly during the merge.  Should this test even be here?  Anyway, if it is supposed to be here, it needs to be fixed because there is an array out of bounds occurring, at least on Linux.  But again, maybe this just needs to be deleted.